### PR TITLE
Make tests

### PR DESCRIPTION
**Pull request by jpweist**
this PR adds test for classes. 
start on the player class 
all test are passing or skipped.
all skipped test are methods or related to methods.

@hannahhch, @khalidwilliams, @brittanystoroz

### CODE REVIEW

**Comment by monstaro** [comment]
Lookin' good!

File: test/game-test.js [hunk 1]
```diff
@@ -1,9 +1,71 @@
...
+
+  it('should hold turn current player', function() {
+    expect(game1.turn.player.name).to.equal('Steve')
+  });
+
+  it('should hold the turn answers', function() {
+    expect(game1.turn.answers).to.deep.equal({ answer: 'Alarm Clock', respondents: 34, surveyId: 3 })
+  });
+
+  it.skip('should have a method to to get surveys', function() {
```
Comment (khalidwilliams) [inline test/game-test.js]: These properties should only be tested in the `Turn-test` file

File: test/game-test.js [hunk 2]
```diff
@@ -1,9 +1,71 @@
...
+  it('should hold round.player1 score', function() {
+    expect(game1.round.player1.score).to.equal(0);
+  });
+
+  it('should hold round.player2 name', function() {
+    expect(game1.round.player2.name).to.equal('Jeff');
+  });
+
+  it('should hold round.player2 score', function() {
+    expect(game1.round.player2.score).to.equal(0);
```
Comment (khalidwilliams) [inline test/game-test.js]: These properties only need to be tested in the `Round-test` file

File: test/round-test.js [hunk 3]
```diff
@@ -1,8 +1,59 @@
...
+    surveys1 = { id: 1, question: 'If You Drew Homer Simpson’s Name In A Secret Santa Exchange, What Would You Buy Him?' };
+    answers1 = { answer: 'Alarm Clock', respondents: 34, surveyId: 3 };
+    playerT1 = new Player('Steve');
+    playerT2 = new Player('Jeff');
+    round1 = new Round(surveys1, playerT1, playerT2, 1);
+
+  });
+
+  it('should hold the surveys data', function() {
+    expect(round1.surveys).to.deep.equal({ id: 1, question: 'If You Drew Homer Simpson’s Name In A Secret Santa Exchange, What Would You Buy Him?' });
```
Comment (khalidwilliams) [inline test/round-test.js]: The logic about a round holding surveys is already tested! We don't need to reassert that these properties exist on a Round that's stored in the Game.

**Pull request merged**